xml/System/String.xml

```diff
@@ -6368,9 +6368,21 @@ String 'This is a string.' in domain 'NewDomain': 75CC8236
       <Docs>
         <param name="value">To be added.</param>
```

Comment:
missing the param descriptions

Reply:
Sorry about that, @mairaw. I've added them.